src/observability/kubearmor.go

```diff
@@ -278,12 +279,23 @@ func GetKubearmorSummaryData(req *opb.Request) ([]types.SysObsProcFileData, []ty
 				Count:       uint32(ss.Count),
 				UpdatedTime: t.Format(time.UnixDate),
 			})
+		} else if ss.Operation == types.OpTypeSyscall {
+			//ExtractsyscallData
+			sysCallData = append(sysCallData, types.SysObsSycallData{
+				ParentProcess: ss.Source,
+				ChildProcess:  ss.Destination,
+				Syscall:       ss.Syscall,
+				Parameters:    ss.Parameters,
+				Count:         uint32(ss.Count),
+				UpdatedTime:   t.Format(time.UnixDate),
+				Result:        ss.Action,
```

Comment:
When we were having design conversation for syscall data we come up with `Result` instead of `Status`. Since it's a syscall it's appropriate to call Result or if you have any other thoughts let me know.